### ISSUE TITLE
Build problem with 'enchant' and 'sphinxexts.taglist'

### ISSUE THREAD

**Issue by m3y54m** [Pelagicore/software-factory#192]
I've followed the instructions in README.md to build the documentation for the first time but I encountered the following error after entering the `make` command:

```
ImportError: The 'enchant' C library was not found and maybe needs to be installed.
```

I installed echant library using this command `sudo apt-get install libenchant1c2a` which solved the error above but this time another error appeared:

```
Extension error:
Could not import extension sphinxexts.taglist (exception: cannot import name 'NoUri' from 'sphinx.environment' (/home/mp/.local/lib/python3.8/site-packages/sphinx/environment/__init__.py))
```

**Comment by thenor** [comment]
Hello @m3y54m. I don't have any build issues. There might be a dependency missing in the README. I can see that I have pyenchant installed. Could you do a "sudo pip check pyenchant" to see if you have this installed and if you do not could you try to install it? 
We need to add it as a dependency if this works, unless @rpannek has another suggestion.

**Comment by rpannek** [comment]
A good start is to use the Dockerfile to build a docker image with all the dependencies and then doing the build inside of that docker image. This way you can be sure that it's not some specific dependency on your computer which is missing.

**Comment by thenor** [comment]
> A good start is to use the Dockerfile to build a docker image with all the dependencies and then doing the build inside of that docker image. This way you can be sure that it's not some specific dependency on your computer which is missing.

Yes, and there is enchant in the list of packages to install in Dockerfile:
```
apt-get update && apt-get install -y \
	cmake \
	git \
	locales \
	python3-pip \
	enchant \
	plantuml
```

**Comment by thenor** [comment]
But still, the README should be updated with this information so that it's clear. It seems the list of deps in the README and the Dockerfile have diverged.

**Comment by thenor** [comment]
It seems we have the same error "NoUri" in our CI environment.
